gunicorn_config.py

```diff
@@ -24,12 +20,8 @@
 if not running_dev():
     # Saves memory in the worker process, but breaks --reload
     preload_app = True
-    # avoid restarting gunicorn and leave it to pod restarts to handle memory leaks
-    max_requests = 0
     graceful_timeout = 800  # if a restart must happen then let it be graceful
     keepalive = 60  # specifically this should be a value larger then nginx setting
-    # ref: https://github.com/benoitc/gunicorn/issues/1978
-    max_requests_jitter = 0
```

Comment:
as per the ref ... we set this as a guard for max_requests_jitter logic to never engage

Reply:
while tests revealed that this setting indeed had no behaviour I thought it best to explicitly set to 0 with code comment referring to additional uncontained behaviour regardless of max_requests=0 setting

Reply:
checking, yes I think at the time ... we had older gunicorn then its been updated since (I think I or someone else did that) ... thanks lets remove it